### DISCUSSION TITLE
LiXee converter

### DISCUSSION THREAD

**Discussion by Matriciel** [Koenkk/zigbee2mqtt#13829] [Devices]
Hi,
I use LiXee TIC module with Z2M 1.27.2 and i have received an OTA for this module.
After upgrade, i have theses errors and no reports:
`error 2022-09-02 22:07:02No converter available for 'current_price' ("")
error 2022-09-02 22:07:02No converter available for 'current_index_tarif' ("")
error 2022-09-02 22:07:02No converter available for 'meter_serial_number' ("")
error 2022-09-02 22:07:02No converter available for 'average_rms_voltage_meas_period' ("")
error 2022-09-02 22:07:02No converter available for 'current_tier1_summ_delivered' ("")
error 2022-09-02 22:07:02No converter available for 'message1' ("")
error 2022-09-02 22:07:02No converter available for 'current_tier2_summ_delivered' ("")
error 2022-09-02 22:07:02No converter available for 'message2' ("")
error 2022-09-02 22:07:02No converter available for 'current_tarif' ("")
error 2022-09-02 22:07:02No converter available for 'relais' ("")
error 2022-09-02 22:07:02No converter available for 'apparent_power' ("")
error 2022-09-02 22:07:02No converter available for 'software_revision' ("")
error 2022-09-02 22:07:02No converter available for 'current_date' ("")
error 2022-09-02 22:07:02No converter available for 'rms_voltage' ("")
error 2022-09-02 22:07:02No converter available for 'current_summ_delivered' ("")
error 2022-09-02 22:07:02No converter available for 'days_profile_next_calendar' ("")
error 2022-09-02 22:07:02No converter available for 'active_enerfy_out_d01' ("")
error 2022-09-02 22:07:02No converter available for 'active_power_max' ("")
error 2022-09-02 22:07:02No converter available for 'active_enerfy_out_d02' ("")
error 2022-09-02 22:07:02No converter available for 'drawn_v_a_max_n1' ("")
error 2022-09-02 22:07:02No converter available for 'rms_current' ("")
error 2022-09-02 22:07:02No converter available for 'available_power' ("")
error 2022-09-02 22:07:02No converter available for 'days_number_next_calendar' ("")
error 2022-09-02 22:07:02No converter available for 'status_register' ("")
error 2022-09-02 22:07:02No converter available for 'power_threshold' ("")
error 2022-09-02 22:07:02No converter available for 'active_power' ("")
error 2022-09-02 22:07:02No converter available for 'active_power_ph_b' ("")
error 2022-09-02 22:07:02No converter available for 'site_id' ("")`
It seem that information are changed with the new version.
It is needed an Z2M upgrade ?
Thank a lot

**Comment by lelas33** [comment]
Hi,
Exactly the same thing for me.
So many differences after an OTA update is surprising.
